nixos/modules/services/networking/murmur.nix

```diff
@@ -386,9 +394,9 @@ in
 
         r ${config.environment.etc."os-release".source},
         r ${config.environment.etc."lsb-release".source},
-        owner rwk /var/lib/murmur/murmur.sqlite,
-        owner rw /var/lib/murmur/murmur.sqlite-journal,
-        owner r /var/lib/murmur/,
+        owner rwk ${cfg.stateDir}/murmur.sqlite,
+        owner rw ${cfg.stateDir}/murmur.sqlite-journal,
+        owner r ${cfg.stateDir},
```

Comment:
Please add the trailing slash here as this commit is breaking it.

Reply:
I think it has been added in yesterday's commit.

Reply:
The PR consists of two commits. The first breaks the test (by omitting the trailing slash) and the second fixes the issue introduced by the first commit.

Reply:
Thanks for the pointer. I've rebased the commits.